pkg/syncer/volume_health.go

```diff
@@ -88,6 +88,7 @@ func csiGetVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 		if volHealthStatus, ok := volumeIdToHealthStatusMap[volID]; ok {
 			// only update PVC health annotation if the HealthStatus of volume is not "unknown"
 			if volHealthStatus != string(pbmtypes.PbmHealthStatusForEntityUnknown) {
+				log.Infof("SPBM returns volume %q with health status %q", volID, volHealthStatus)
```

Comment:
Wouldn't this fill up the logs in a scaled environment? Why is this log critical?